content/en/reference/exchanges/exchange-integration/index.md

```diff
@@ -294,3 +294,4 @@ Join the [Filecoin Slack](https://filecoinproject.slack.com/signup) and post any
 - [Filecoin Networks](https://status.filecoin.io)
 - [Lotus](https://github.com/filecoin-project/lotus) - the reference implementation for the Filecoin network
 - [Block Explorers]({{< relref "/networks/mainnet/explorers" >}})
+<!--REVIEWED!-->
```

Comment:
Similar to point 1 in https://github.com/filecoin-project/filecoin-docs/pull/1978#discussion_r1191789828, the spell checker is flagging a bunch of function names as spelling errors. I'd like to go back through this page, try reformatting each function using ``, and re-run checks